application/common/models/User.php

```diff
@@ -1115,4 +1111,34 @@ public static function numberWithOneMfaNotBackupCodes()
         ")->queryOne();
         return $result['COUNT(*)'];
     }
+
+    protected function isExpired(): bool
+    {
+        return $this->expires_on != null && MySqlDateTime::isBefore($this->expires_on, time());
```

Comment:
Did you intentionally change `!==` to `!=` in the `expires_on` check?

Reply:
Nope. Changed back to `!==`.